### PR TITLE
[DOCS-9184] Update Datadog Address prerequesite

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

- Adds a clarification.

[DOCS-9184](https://datadoghq.atlassian.net/browse/DOCS-9184)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-9184]: https://datadoghq.atlassian.net/browse/DOCS-9184?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md [hunk 1]
```diff
@@ -233,7 +233,7 @@ Follow the instructions for the cloud provider you are using to archive your log
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```
Comment (maycmlee) [inline content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md:236]: After we finalize the wording here, I'll update all Datadog Agent setup docs (eg. for split logs, log volume control, etc).

**Review by ckelner — APPROVED** [review]
LGTM!

File: content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md (hunk 1 above)
Comment (maycmlee) [reply content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md:236]: updated

Comment (rtrieu) [inline content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md:236]:
```suggestion
1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

File: content/en/observability_pipelines/set_up_pipelines/dual_ship_logs/datadog_agent.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ Enter the following information based on your selected logs destination.
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```
Comment (rtrieu) [inline content/en/observability_pipelines/set_up_pipelines/dual_ship_logs/datadog_agent.md:163]:
```suggestion
1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

File: content/en/observability_pipelines/set_up_pipelines/generate_metrics/datadog_agent.md [hunk 3]
```diff
@@ -158,7 +158,7 @@ Enter the following information based on your selected logs destination.
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```
Comment (rtrieu) [inline content/en/observability_pipelines/set_up_pipelines/generate_metrics/datadog_agent.md:161]:
```suggestion
1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

File: content/en/observability_pipelines/set_up_pipelines/log_enrichment/datadog_agent.md [hunk 4]
```diff
@@ -160,7 +160,7 @@ Enter the following information based on your selected logs destination.
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```
Comment (rtrieu) [inline content/en/observability_pipelines/set_up_pipelines/log_enrichment/datadog_agent.md:163]:
```suggestion
1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

File: content/en/observability_pipelines/set_up_pipelines/log_volume_control/datadog_agent.md [hunk 5]
```diff
@@ -160,7 +160,7 @@ Enter the following information based on your selected logs destination.
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```
Comment (rtrieu) [inline content/en/observability_pipelines/set_up_pipelines/log_volume_control/datadog_agent.md:163]:
```suggestion
1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

File: content/en/observability_pipelines/set_up_pipelines/sensitive_data_redaction/datadog_agent.md [hunk 6]
```diff
@@ -162,7 +162,7 @@ Enter the following information based on your selected logs destination.
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```
Comment (rtrieu) [inline content/en/observability_pipelines/set_up_pipelines/sensitive_data_redaction/datadog_agent.md:165]:
```suggestion
1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

File: content/en/observability_pipelines/set_up_pipelines/split_logs/datadog_agent.md [hunk 7]
```diff
@@ -173,7 +173,7 @@ Enter the following information based on your selected logs destination.
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```
Comment (rtrieu) [inline content/en/observability_pipelines/set_up_pipelines/split_logs/datadog_agent.md:176]:
```suggestion
1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

File: layouts/shortcodes/observability_pipelines/prerequisites/datadog_agent.en.md [hunk 8]
```diff
@@ -2,7 +2,7 @@ You already have the Datadog Agent installed to collect and route your logs to [
 
 You have the following information available:
 - The Datadog Agent address, including the port.
-    -  The Observability Pipelines Worker listens to this socket address to receive logs from the Datadog Agent. 
+    -  The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```
Comment (rtrieu) [inline layouts/shortcodes/observability_pipelines/prerequisites/datadog_agent.en.md:5]:
```suggestion
    -  The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

**Review by rtrieu — APPROVED** [review]
Minor grammatical suggestion.

File: content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md (hunk 1 above)
Comment (maycmlee) [reply content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md:236]: Thanks @rtrieu! I'd like to keep second part with the first part of the sentence, what do you think of this?

> The Observability Pipelines Worker listens to this address and port for incoming logs from the Datadog Agent. For example, `0.0.0.0:<port_number>`.

Comment (rtrieu) [reply content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md:236]: I think that works even better!

**Review by rtrieu — APPROVED** [review]

**Pull request merged**